src/Sylius/Bundle/ResourceBundle/Form/Type/ResourceToHiddenIdentifierType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\Form\Type;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Sylius\Bundle\ResourceBundle\Form\DataTransformer\ResourceToIdentifierTransformer;
+use Sylius\Component\Resource\Metadata\MetadataInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Exception\LogicException;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Joseph Bielawski <stloyd@gmail.com>
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class ResourceToHiddenIdentifierType extends ResourceToIdentifierType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->addModelTransformer(new ResourceToIdentifierTransformer($this->repository, $options['identifier']))
```

Comment:
There was another model trannsformer set other way earlier:
```php
$builder->addModelTransformer(
    new IdentifierToResourceTransformer($this->repository, $options['identifier'])
);
```

It would be nice to stick to one convention.